### PR TITLE
Initialize repository

### PR DESCRIPTION
**Pull request by mihailradkov**
Changes

- Added CI pipeline with GitHub actions for analyzing the modules
- Added issue and PR templates
- Added changelog, code owners, contributors guide etc.
- Added initial skeleton module

### CODE REVIEW

File: .github/ISSUE_TEMPLATE /bug_report.md  [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a bug report to help us improve.
+title: ''
+labels: bug
```
Comment (viktor-ribchev) [inline .github/ISSUE_TEMPLATE /bug_report.md :5]: Priority?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: ci
```
Comment (mihailradkov) [inline .github/workflows/ci.yml:1]: Change to `Continuous Integration`

Comment (tonyKunchev) [reply .github/workflows/ci.yml:1]: Just FYI, we have limited number of free `GitHub Workflow` runs per month.

We've other repositories, which also are using GitHub Actions. Not sure how intensive you are planning to use them, but keep in mind the above.

Comment (mihailradkov) [reply .github/workflows/ci.yml:1]: Not much, just for simple linting and security checks.

**Comment by yaskoo** [comment]
We should probably license this somehow.

**Review by yaskoo — APPROVED** [review]
LGTM

**Pull request merged**